### PR TITLE
Address prefix and  modifications related to close CDP. also includes PR #28, query CDP changes.

### PR DESCRIPTION
**Pull request by dheerajkd30**

### CODE REVIEW

File: x/cdp/client/rest/tx.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ func createRepay(ctx client.Context) http.HandlerFunc {
 	}
 }
 
-func createLiquidate(ctx client.Context) http.HandlerFunc {
+func createCloseCDP(ctx client.Context) http.HandlerFunc {
```
Comment (prashantkrv2020) [inline x/cdp/client/rest/tx.go]: it should be just closeCDP

File: x/cdp/client/rest/rest.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ func registerTxRoutes(ctx client.Context, router *mux.Router) {
 		Methods("POST")
 	router.HandleFunc("/repay", createRepay(ctx)).
 		Methods("POST")
-	router.HandleFunc("/liquidate", createLiquidate(ctx)).
+	router.HandleFunc("/closecdp", createCloseCDP(ctx)).
```
Comment (prashantkrv2020) [inline x/cdp/client/rest/rest.go]: better path would be /cdp/close

File: x/cdp/keeper/grpc_query.go [hunk 3]
```diff
@@ -18,26 +19,64 @@ func (k Keeper) QueryCDP(context context.Context, request *types.QueryCDPRequest
...
+	)
+
+	ownerAddrs, err := sdk.AccAddressFromBech32(request.Owner)
+	if err != nil {
+		return &types.QueryCDPsResponse{}, err
+	}
+
+	ownerCDPList, found := k.GetOwnerCDPList(ctx, ownerAddrs)
+	if !found {
+		return &types.QueryCDPsResponse{}, status.Error(codes.NotFound, "no cdps created for this account")
```
Comment (prashantkrv2020) [inline x/cdp/keeper/grpc_query.go]: if the owner has no then that's not an error, just send an empty array.

**Comment by bsrinivas8687** [comment]
This perticular CDP module is not compatible with the asset module. We have to refactor it. I have done some level of progress on branch `rebase-cdp-module`. Please have a look at it. We are not developing another K**a.

**Comment by dheerajkd30** [comment]
> This perticular CDP module is not compatible with the asset module. We have to refactor it. I have done some level of progress on branch `rebase-cdp-module`. Please have a look at it. We are not developing another K**a.

anything specific in this PR made you think of this ? or are you referring to CDP module as a whole? As it includes few extra query functions and address prefix changes.

**Comment by bsrinivas8687** [comment]
> > This perticular CDP module is not compatible with the asset module. We have to refactor it. I have done some level of progress on branch `rebase-cdp-module`. Please have a look at it. We are not developing another K**a.
> 
> anything specific in this PR made you think of this ? or are you referring to CDP module as a whole? As it includes few extra query functions and address prefix changes.

This is specific to whole CDP module

**Comment by bsrinivas8687** [comment]
You can make another PR with correct Address prefixes

**Pull request closed without merging**